<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="reel1">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="reel2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="reel3">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>170</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>36</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="PULLButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>PULL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bet1">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>X1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bet2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>X2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bet3">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>370</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>X3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>AAA- x1</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Current Balance</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="balance">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>71</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>BBB- x2</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>CCC -x3</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DDD -x4</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>250</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>EEE  -x5</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>FFF  - x6</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>310</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>GGG -x7</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>330</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ABC -x8</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>360</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>DEF -x9</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Any other combo -x0</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
